### ISSUE TITLE
[ubuntu] Auto-Configuration Warning: 'TMP' environment variable is not set, using 'C:\Windows\Temp' as default

### ISSUE THREAD

**Issue by guysoft** [tensorflow/tensorflow#22654]
If you open a GitHub issue, here is our policy:

### System information
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**:
no

- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:
Ubuntu 18.04
- **Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device**:
N/A
- **TensorFlow installed from (source or binary)**:
release tag v1.110
- **TensorFlow version (use command below)**:
- **Python version**:
3.6
- **Bazel version (if compiling from source)**:
0.17.2
- **GCC/Compiler version (if compiling from source)**:
4.9
- **CUDA/cuDNN version**:
9.1
- **GPU model and memory**:
GeForce 940M/PCIe/SSE2 2GB
- **Exact command to reproduce**:
Commands below. ``./configure`` then;
```
bazel build --config=opt --config=cuda //tensorflow/tools
```

You can collect some of this information using our environment capture script:

https://github.com/tensorflow/tensorflow/tree/master/tools/tf_env_collect.sh
[tf_env.txt](https://github.com/tensorflow/tensorflow/files/2435301/tf_env.txt)

You can obtain the TensorFlow version with
None, 

### Describe the problem
Getting on compliation error:
Auto-Configuration Warning: 'TMP' environment variable is not set, using 'C:\Windows\Temp' as default

### Source code / logs

```
~/tmp/tensorflow$ bazel build --config=opt --config=cuda //tensorflow/tools/pip_package:build_pip_package
Starting local Bazel server and connecting to it...
WARNING: The following configs were expanded more than once: [cuda]. For repeatable flags, repeats are counted twice and may lead to unexpected behavior.
Loading: 0 packages loaded
Loading: 0 packages loaded
    currently loading: tensorflow/tools/pip_package
DEBUG: /home/guy/.cache/bazel/_bazel_guy/0d29558304d115cf217e115548450924/external/bazel_tools/tools/cpp/lib_cc_configure.bzl:115:5: 
Auto-Configuration Warning: 'TMP' environment variable is not set, using 'C:\Windows\Temp' as default
```

my configure:
```
$ ./configure                  
Extracting Bazel installation...
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by com.google.protobuf.UnsafeUtil (file:/home/guy/.cache/bazel/_bazel_guy/install/792a28b07894763eaa2bd870f8776b23/_embedded_binaries/A-server.jar) to field java.lang.String.value
WARNING: Please consider reporting this to the maintainers of com.google.protobuf.UnsafeUtil
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
WARNING: --batch mode is deprecated. Please instead explicitly shut down your Bazel server using the command "bazel shutdown".
You have bazel 0.17.2 installed.
Please specify the location of python. [Default is /usr/bin/python]: /usr/bin/python3


Found possible Python library paths:
  /usr/lib/python3/dist-packages
  /usr/local/lib/python3.6/dist-packages
Please input the desired Python library path to use.  Default is [/usr/lib/python3/dist-packages]

Do you wish to build TensorFlow with jemalloc as malloc support? [Y/n]: 
jemalloc as malloc support will be enabled for TensorFlow.

Do you wish to build TensorFlow with Google Cloud Platform support? [Y/n]: n
No Google Cloud Platform support will be enabled for TensorFlow.

Do you wish to build TensorFlow with Hadoop File System support? [Y/n]: n
No Hadoop File System support will be enabled for TensorFlow.

Do you wish to build TensorFlow with Amazon AWS Platform support? [Y/n]: n
No Amazon AWS Platform support will be enabled for TensorFlow.

Do you wish to build TensorFlow with Apache Kafka Platform support? [Y/n]: n
No Apache Kafka Platform support will be enabled for TensorFlow.

Do you wish to build TensorFlow with XLA JIT support? [y/N]: y
XLA JIT support will be enabled for TensorFlow.

Do you wish to build TensorFlow with GDR support? [y/N]: n
No GDR support will be enabled for TensorFlow.

Do you wish to build TensorFlow with VERBS support? [y/N]: 
No VERBS support will be enabled for TensorFlow.

Do you wish to build TensorFlow with nGraph support? [y/N]: 
No nGraph support will be enabled for TensorFlow.

Do you wish to build TensorFlow with OpenCL SYCL support? [y/N]: 
No OpenCL SYCL support will be enabled for TensorFlow.

Do you wish to build TensorFlow with CUDA support? [y/N]: y
CUDA support will be enabled for TensorFlow.

Please specify the CUDA SDK version you want to use. [Leave empty to default to CUDA 9.0]: 9.1


Please specify the location where CUDA 9.1 toolkit is installed. Refer to README.md for more details. [Default is /usr/local/cuda]: /usr


Please specify the cuDNN version you want to use. [Leave empty to default to cuDNN 7.0]: 7.1


Please specify the location where cuDNN 7 library is installed. Refer to README.md for more details. [Default is /usr]: /usr


Do you wish to build TensorFlow with TensorRT support? [y/N]: 
No TensorRT support will be enabled for TensorFlow.

Please specify the NCCL version you want to use. If NCCL 2.2 is not installed, then you can use version 1.3 that can be fetched automatically but it may have worse performance with multiple GPUs. [Default is 2.2]: 1.3


Please specify a list of comma-separated Cuda compute capabilities you want to build with.
You can find the compute capability of your device at: https://developer.nvidia.com/cuda-gpus.
Please note that each additional compute capability significantly increases your build time and binary size. [Default is: 3.5,7.0]: 5.0


Do you want to use clang as CUDA compiler? [y/N]: 
nvcc will be used as CUDA compiler.

Please specify which gcc should be used by nvcc as the host compiler. [Default is /usr/bin/x86_64-linux-gnu-gcc-7]: /usr/bin/x86_64-linux-gnu-gcc-4.9


Do you wish to build TensorFlow with MPI support? [y/N]: 
No MPI support will be enabled for TensorFlow.

Please specify optimization flags to use during compilation when bazel option "--config=opt" is specified [Default is -march=native]: 


Would you like to interactively configure ./WORKSPACE for Android builds? [y/N]: 
Not configuring the WORKSPACE for Android builds.

Preconfigured Bazel build configs. You can use any of the below by adding "--config=<>" to your build command. See tools/bazel.rc for more details.
        --config=mkl            # Build with MKL support.
        --config=monolithic     # Config for mostly static monolithic build.
Configuration finished
```

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Mobile device

**Comment by guysoft** [comment]
Fixed, - **Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device**: 
N/A

**Comment by zishanahmed08** [comment]
i faced the same issue. Any fixes yet?

**Comment by wt-huang** [comment]
@guysoft You can use CUDA 9.2/cuDNN 7.1 and Bazel 1.15.0 to install TensorFlow.

**Comment by hanysz** [comment]
I had the same issue, but it inexplicably went away after rebooting my computer (very sad, that's the type of behaviour I expect from a Windows system, not linux).  Using CUDA 9.1, cuDNN 7.2.1, Bazel 0.17.2 here, with Ubuntu 18.04.

**Comment by wt-huang** [comment]
That happens sometimes, but much less frequently.

**Comment by bzamecnik** [comment]
In my case (bazel-1.19.1 / tf-1.12) upgrading to bazel-1.19.2 fixed this issue and also #23719.

**Comment by Tahirhan** [comment]
> In my case (bazel-1.19.1 / tf-1.12) upgrading to bazel-1.19.2 fixed this issue and also #23719.

also fixed my issue

**Comment by chrisspen** [comment]
@bzamecnik How did you build Tensorflow with Bazel 1.19.2? If I try to build the current release of Tensorflow with Bazel>=1, the installer errors and tells me I must downgrade to Bazel 0.26.1 or earlier.
